src/components/Comments.jsx

```diff
@@ -25,6 +29,31 @@ function Comments(props) {
       });
   }, [id, comments]);
```

Comment:
I would remove comments from the dependency array here - it reverses your work of spreading the new comment to the TOP of the list as it then re-fetches all of them anyway.
This means that the new comment is now at the bottom and not clear that it has worked to the user